### DISCUSSION TITLE
Don't know why this issue was closed

### DISCUSSION THREAD

**Discussion by carlosmintfan** [twbs/bootstrap#40489] [General] [answered]
Please reopen this if you closed it accidentally. 😃

_Originally posted by @carlosmintfan in https://github.com/twbs/bootstrap/issues/40438#issuecomment-2126209312_
            
Please check original issue and reopen if necessary.

**Comment by julien-deramond** [comment, marked as answer]
Hello @carlosmintfan 
I've moved your issue to a discussion here.

I'm not sure if there's an issue in https://github.com/twbs/bootstrap/issues/40438, or I don't understand exactly what was reported.
In our [Nav and tabs documentation](https://getbootstrap.com/docs/5.3/components/navs-tabs/):
* The entire first part of the documentation shows the effect of classes and DOM structure on the rendering, explains the concepts, but doesn't do anything dynamically. The developer has to switch the elements programmatically (e.g. `.active` class, `aria-selected`, etc.)
* The only dynamic part is explained in [JavaScript behavior](https://getbootstrap.com/docs/5.3/components/navs-tabs/#javascript-behavior), and requires JavaScript.

If you think there's a use case where it doesn't work, you can create a new issue and integrate a reproducible live example with CodePen, StackBlitz, or any other tool, to make that more understandable :)

**Reply by carlosmintfan** [reply]
All right, I didn't test, and yeah, it's all the right behaviour, I just opened it in case the original issue was closed accidentally or whyever, the author just closed it without explaining...

**Comment by jipexu** [comment]
Hi
Yes no issue there. 
For example the concept explained by @julien-deramond is the same for example in the navbar doc where the link are dead of course.....
